contracts/Edition.sol

```diff
@@ -88,25 +90,46 @@ contract Edition is
         animationUrl = _animationUrl;
         imageUrl = _imageUrl;
 
-        uint64 endOfMintPeriod;
+        uint64 _endOfMintPeriod;
```

Comment:
is this a leftover from the previous pr?

Reply:
yeah I missed this spot 🙈